maskrcnn_benchmark/modeling/backbone/resnet.py

```diff
@@ -230,24 +260,25 @@ def __init__(
             stride=stride_1x1,
             bias=False,
         )
-        self.bn1 = FrozenBatchNorm2d(bottleneck_channels)
+        self.bn1 = norm_func(bottleneck_channels)
         # TODO: specify init for the above
 
         self.conv2 = Conv2d(
             bottleneck_channels,
             bottleneck_channels,
             kernel_size=3,
             stride=stride_3x3,
-            padding=1,
+            padding=dilation, # dilation * (kernel_size - 1) // 2,
```

Comment:
just checking if this comment should be removed or not?

Reply:
it can be removed, but can be a explanation for using dilation param. i will remove it.

Reply:
oh ok, it was not clear to me, it looked like you were trying to use one or the other